src/main.rs

```diff
@@ -75,3 +80,11 @@ fn download_tar(target: &str) -> Result<TarArchive, CliError> {
     Ok(tar::Archive::new(decoded_response))
 }
 
+
+fn store_token_in_home(token: String) -> std::io::Result<()>{
+    let home = env::var("HOME").unwrap();
+    let config_path = format!("{}/.sharpener-config", home);
```

Comment:
Use the `std::path` library instead of manually joining paths. This will deal with any corner cases that can happen when joining two paths.
```rust
use std::path::PathBuf;
let home = env::var("HOME").map(PathBuf::from).unwrap();
let config_path = home.join(".sharpener-config");
```